app/models/rental.rb

```diff
@@ -0,0 +1,18 @@
+class Rental < ApplicationRecord
+  belongs_to :customer 
+  belongs_to :video
+
+  validates :customer_id, presence: true
+  validates :video_id, presence: true
+
+  def update_check_out_dates
+    self.check_out_date = Date.today
+    self.due_date = Date.today + 7
+  end
```

Comment:
It's confusing that this method doesn't save but your other update methods do.

```suggestion
  def update_check_out_dates
    self.check_out_date = Date.today
    self.due_date = Date.today + 7
    
    return self.save
  end
```